libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKDevInfoViewController.m

```diff
@@ -120,7 +119,6 @@ - (UINavigationBar*) createNavBar
 - (UITableView*) createTableView
 {
     UITableView *infoTable = [[UITableView alloc] initWithFrame:CGRectZero];
-    infoTable.backgroundColor = [UIColor lightGrayColor];
```

Comment:
Took this out because it was making the notch a different color than the rest of the tableview, could keep it and have dark/light versions if wanted